product/views.py

```diff
@@ -1,3 +1,94 @@
-from django.shortcuts import render
+from django.http                    import HttpResponse, JsonResponse
+from django.views                   import View
+from django.db.models               import Count, F
 
-# Create your views here.
+import json
+
+from .models          import Product, Category, SubCategory
+
+class ListView(View):
+    def get(self, request):
+        page = int(request.GET.get("page",1))
+        category_code = request.GET.get("cateCd")
+        
+        list_count = 28
+        limit = list_count * page
+        offset = limit - list_count
+        all_products = Product.objects.all().prefetch_related(
+                "thumbnail_image"
+                ).select_related(
+                        "sub_category"
+                        )[offset:limit]
+
+         # Category 기준 카운트
+        rename_list = Product.objects.annotate(
+                category_name = F("sub_category__category__name"),
+                category_code = F("sub_category__category__code"),
+                )
+        total_ctgry = rename_list.values(
```

Comment:
```suggestion
        total_category = rename_list.values(
```

변수명 줄여쓰는 습관은 지양해주세요. 올바른 변수명 작성하는 습관을 지향해주세요.